README.md

```diff
@@ -377,6 +377,7 @@ These settings can be changed at any time.
 [`admin`](https://github.com/bottlerocket-os/bottlerocket-admin-container) and [`control`](https://github.com/bottlerocket-os/bottlerocket-control-container) are our default host containers, but you're free to change this.
 Beyond just changing the settings above to affect the `admin` and `control` containers, you can add and remove host containers entirely.
 As long as you define the three fields above -- `source` with a URI, and `enabled` and `superpowered` with true/false -- you can add host containers with an API call or user data.
+You can optionally define a `user-data` field with arbitrary base64-encoded data, which will be made available in the container at `/.bottlerocket/host-containers/$HOST_CONTAINER_NAME/user-data`.
```

Comment:
Do we want to add a few more words around the fact that this data can be whatever you want it to be as long as whatever is running in the host container understands how to use it?  We just want to make it completely clear that this isn't in reference to, or tied to cloud-init or any other similar system.